### ISSUE TITLE
Drop-In's input TextView gets messed up when hitting Return key to enter new lines

### ISSUE THREAD

**Issue by KBog** [intercom/intercom-ios#285]
Please refer to the attached gif.

## Version info
- Intercom for iOS version: 3.2.9
- iOS version: 10.3

## Expected behavior
- The UITextView shouldn't jump all over the place when hitting ENTER to add, or BACKSPACE to delete characters.

## Actual behavior
- The TextView and surrounding views are getting messed up.

## Steps to reproduce
 1. Trigger the Intercom drop in to show.
 2. Type, then hit Enter

## Logs
![intercom_textview_issue](https://user-images.githubusercontent.com/2876579/28799048-c49a6eb4-75fb-11e7-8a36-81de9d5c7a20.gif)

**Comment by skylerwshaw** [comment]
@KBog, thanks for sending this in. Can you please share any dependencies that may affect keyboard activities in your app. Are you using any components to customize keyboard behavior such as `DAKeyboardControl` or `IQKeyboardManager`?

**Comment by KBog** [comment]
I'm using `IQKeyboardManager`. Is that not supported?

**Comment by skylerwshaw** [comment]
Please see: https://github.com/intercom/intercom-ios/issues/156#issuecomment-236141456
tl;dr: You can work around `IQKeyboardManager`'s interference here.

Definitely message back here or within the Intercom in-app messenger within your Intercom dashboard if you have any other questions on this. 😄
